src/upload-tests.proj

```diff
@@ -115,14 +115,28 @@
     </ItemGroup>
   </Target>
 
+  <!-- Zips up the tools used for UAP-style execution.  TODO:  NuPkg-ify these tools. -->
```

Comment:
I suspect we don't need this anymore since during build I am placing this in the testhost folder next to the uaplayout folder.

Will keep this in here and revisit once we have the package for the runner and launcher created.